Public/robots.txt

```diff
@@ -1 +1,25 @@
 Sitemap: https://swiftpackageindex.com/sitemap.xml
+
+User-agent: Bytespider
+Disallow: /
+
+User-agent: GPTBot
+Disallow: /
+
+User-agent: ClaudeBot
+Disallow: /
+
+User-agent: anthropic-ai
+Disallow: /
+
+User-agent: Applebot
+Disallow: /
+
+User-agent: Google-Extended
+Disallow: /
+
+User-agent: Cohere-ai
+Disallow: /
+
+User-agent: Seekr
+Disallow: /
```

Comment:
I'll go ahead and enable the CF block to see if this stop the crawl traffic.

Reply:
Actually, I'll give it until this afternoon. We updated `robots.txt` at noon yesterday and Google [update their view of it every 24h](https://developers.google.com/search/docs/crawling-indexing/robots/submit-updated-robots-txt#refresh-googles-robots.txt-cache). This might be true for the other crawlers so let's see if the initial `robots.txt` has any effect on its own after 24h.